conductor_api/src/dpki_instance.rs

```diff
@@ -40,6 +40,12 @@ fn dpki_cap_request(
     ))
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone, DefaultJson)]
```

Comment:
I think you don't need this anymore right?

Reply:
:+1: